pkg/utils/common/controller.go

```diff
@@ -500,3 +502,99 @@ func (s *SuiteController) ApplicationGitopsRepoExists(devfileContent string) wai
 		return s.Github.CheckIfRepositoryExist(gitOpsRepoURL), nil
 	}
 }
+
+//CreateServiceAccount create ServiceAccount
```

Comment:
Can the comment be a teeny bit more english?

Reply:
Done